libs/ktem/ktem/index/file/pipelines.py

```diff
@@ -366,10 +369,11 @@ def insert_chunks_to_vectorstore():
                 chunks = to_index_chunks[start_idx : start_idx + chunk_size]
                 self.handle_chunks_vectorstore(chunks, file_id)
                 n_chunks += len(chunks)
-                yield Document(
-                    f" => [{file_name}] Created embedding for {n_chunks} chunks",
-                    channel="debug",
-                )
+                if self.VS:
```

Comment:
converting this one to `self.VectorStore` doesn't hurt nowadays as we have autocompletion